tests/cupy_tests/test_cusolver.py

```diff
@@ -177,3 +177,56 @@ def test_syevjBatched(self):
 
         self.assertEqual(v.shape, a.shape)
         self.assertEqual(w.shape, a.shape[:-1])
+
+
+@testing.parameterize(*testing.product({
+    'dtype': ['float32', 'float64', 'complex64', 'complex128'],
```

Comment:
Could you align the style with other test classes in the file?
```suggestion
    'dtype': [np.float32, np.float64, np.complex64, np.complex128],
```
(or fix the others.)